mlcomp/parallelm/components/restful_component.py

```diff
@@ -213,8 +215,8 @@ def run(cls, port, model_path):
         UwsgiBroker._application.run(host='localhost', port=port)
 
     # NOTE: do not rename this route or over-ride it
-    @FlaskRoute('/{}'.format(RestfulConstants.STATS_ROUTE))
-    #@FlaskRoute('/statsinternal')
+    #@FlaskRoute('/{}'.format(RestfulConstants.STATS_ROUTE))
+    @FlaskRoute('/statsinternal')
```

Comment:
I guess that the endpoint name `statsinternal` is used also somewhere else to fetch the data. Why not using the previous string format?

Reply:
The previous was not working.. for some reason only when I provide the actual string it is working.

Reply:
Interesting. I'll try to test it sometime soon.